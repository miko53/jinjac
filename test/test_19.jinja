Test if statement
{% if gre > 100 %}
This text must appear with value {{ myNiceDouble }} 
{% else %}
DSFDSF
{% endif %}
mlcdkmlsfk
{% if users %}
User exists {{ users[0] }}
{% endif %}
{% if do_not_exist %}
Must not exist 
{% else %}
It seems not exist !!
{% endif %}
