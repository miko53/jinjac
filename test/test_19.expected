Test if statement

This text must appear with value 0.156844 

